modules/ld/shacl-topquadrant-validator/src/main/scala/ch/epfl/bluebrain/nexus/commons/shacl/topquadrant/Vocabulary.scala

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.bluebrain.nexus.commons.shacl.topquadrant
+
+import ch.epfl.bluebrain.nexus.rdf.Node.IriNode
+import ch.epfl.bluebrain.nexus.rdf.syntax.node.unsafe._
+import org.apache.jena.rdf.model.{Property, ResourceFactory}
+
+object Vocabulary {
+
+  /**
+    * Shacl vocabulary.
+    */
+  object sh {
+    val base             = "http://www.w3.org/ns/shacl#"
+    val ValidationReport = url"${base}ValidationReport"
+    val conforms         = url"${base}conforms"
+  }
+
+  /**
+    * Nexus shacl vocabulary.
+    */
+  object nxsh {
+    val base          = "https://bluebrain.github.io/nexus/vocabulary/shacl/"
+    val targetedNodes = url"${base}targetedNodes"
+  }
+
+  private[topquadrant] implicit def toProperty(iriNode: IriNode): Property =
```

Comment:
We should start avoiding these implicit conversions as they will phased out in the next scala releases. The recommendation is to have something similar as with the JavaConverters where you need to explicitly call a function on the enrichment class, or place these conversions in the companion object of the type.